common/src/main/java/org/apache/uniffle/common/util/RssUtils.java

```diff
@@ -292,4 +297,38 @@ public static String getMetricNameForHostName(String hostName) {
     }
     return hostName.replaceAll("[\\.-]", "_");
   }
+
+  public static Map<Integer, Roaring64NavigableMap> shuffleBitmapToPartitionBitmap(
```

Comment:
Why do we need `startPartition` and `endPartition`?

Reply:
Because the partition may not have a block, but we need to fill it to avoid NPE when used, See 
```
for (int partitionId = startPartition; partitionId <= endPartition; partitionId++) {
      result.putIfAbsent(partitionId, Roaring64NavigableMap.bitmapOf());
    }
```

Reply:
> Because the partition may not have a block, but we need to fill it to avoid NPE when used, See
> 
> ```
> for (int partitionId = startPartition; partitionId <= endPartition; partitionId++) {
>       result.putIfAbsent(partitionId, Roaring64NavigableMap.bitmapOf());
>     }
> ```

A little weird.

Reply:
Maybe we should have a better name.

Reply:
This comment isn't resolved.

Reply:
remind